demisto_sdk/json_to_outputs/json_to_outputs.py

```diff
@@ -0,0 +1,200 @@
+import yaml
+import argparse
+import json
+import sys
+import dateparser
+from demisto_sdk.common.tools import print_error, print_warning, print_color, LOG_COLORS
+
+"""
+This script parse a Context output into yml formatted file with the context path of each field.
+For example:
+
+{
+    "id":12131,
+    "description":"desc",
+    "summary":"do-not-delete",
+    "created":"2019-03-25T16:13:13.188+0200",
+    "issuetype":{
+        "id":10004,
+        "name":"Bug"
+    },
+    "project":{
+        "id":10001,
+        "key":"SOC",
+        "projectTypeKey":"software"
+    },
+    "status":{
+        "id":10003,
+            "StatusCategory":{
+                "key":"new",
+                "colorName":"blue-gray",
+                "name":"To Do"
+        }
+    }
+}
+
+==>
+
+arguments: []
+name: integration-command
+outputs:
+- contextPath: Demisto.Id
+  description: ''
+  type: Number
+- contextPath: Demisto.Description
+  description: ''
+  type: String
+- contextPath: Demisto.Summary
+  description: ''
+  type: String
+- contextPath: Demisto.Created
+  description: ''
+  type: String
+- contextPath: Demisto.Issuetype.Id
+  description: ''
+  type: Number
+- contextPath: Demisto.Issuetype.Name
+  description: ''
+  type: String
+- contextPath: Demisto.Project.Id
+  description: ''
+  type: Number
+- contextPath: Demisto.Project.Key
+  description: ''
+  type: String
+- contextPath: Demisto.Project.ProjectTypeKey
+  description: ''
+  type: String
+- contextPath: Demisto.Status.Id
+  description: ''
+  type: Number
+- contextPath: Demisto.Status.StatusCategory.Key
+  description: ''
+  type: String
+- contextPath: Demisto.Status.StatusCategory.Colorname
+  description: ''
+  type: String
+- contextPath: Demisto.Status.StatusCategory.Name
+  description: ''
+  type: String
+"""
+
+
+def input_multiline():
+    sentinel = ''  # ends when this string is seen
+    return '\n'.join(iter(input, sentinel))
+
+
+def flatten_json(nested_json, camelize=False):
+    out = {}
+
+    def flatten(x, name=''):
+        # capitalize first letter in each key
+        try:
+            name = name[0].upper() + name[1:] if camelize else name
+        except IndexError:
+            name = name.title() if camelize else name
+
+        if isinstance(x, dict):
+            for a in x:
+                flatten(x[a], name + a + '.')
+        elif isinstance(x, list):
+            for a in x:
+                flatten(a, name[:-1] + '.')
+        else:
+            out[name.rstrip('.')] = x
+
+    flatten(nested_json)
+    return out
+
+
+def jsonise(context_key, value, description=''):
+    return {
+        'contextPath': context_key,
+        'description': description,
+        'type': determine_type(value)
+    }
+
+
+def is_date(val):
+    if isinstance(val, (int, float)) and val > 15737548065 and val < 2573754806500:
```

Comment:
what are those numbers, and why does a number that is bigger then 15737548065 is date?